refactor.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable no-alert */
+/* eslint-disable comma-dangle */
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+class ui {
+  static displaybooks() {
+    const books = store.getbooks();
+    books.forEach((book) => ui.addbooktolist(book));
+  }
+
+  static addbooktolist(book) {
+    const list = document.querySelector('#book-addition');
+    const div = document.createElement('div');
+    div.innerHTML = `
+        <p>${book.title} by ${book.author}</p>
+        <Button class="remove-btn">Remove</Button>
+        <hr>
+        `;
+    list.appendChild(div);
+  }
+
+  static deletebook(el) {
+    if (el.classList.contains('remove-btn')) {
+      el.parentElement.remove();
+    }
+  }
+
+  static clearFields() {
+    document.querySelector('#title').value = '';
+    document.querySelector('#author').value = '';
+  }
+}
+
+class store {
+  static getbooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  static addbook(book) {
+    const books = store.getbooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removebook(title) {
+    const books = store.getbooks();
+    books.forEach((book, index) => {
+      if (book.title === title) {
+        books.splice(index, 1);
+      }
+    });
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+}
+
```

Comment:
- [ ] it would be better to save the books  to local storage after removing one of them cause right now when I remove a book and reload the page it comes back please make sure to fix that